htdocs/expedition/card.php

```diff
@@ -2443,6 +2473,12 @@
 				}
 			}
 
+			// Modify
+			if ($object->statut == Expedition::STATUS_VALIDATED && $user->rights->expedition->creer) {
```

Comment:
No new features are allowed into an already published version. Please send this change into a PR for develop branch